### PR TITLE
Add test jwk to pepper service reading from secure_test_jwk.json 

### PR DESCRIPTION
**Pull request by heliuchuan**
## Description

Adds the test jwk from types/src/jwks/rsa/secure_test_jwk.json.  This cleans up hardcoding of the test jwk and helps us move toward a single source of truth for our test jwks.

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [x] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [x] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [x] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I identified and added all stakeholders and component owners affected by this change as reviewers
- [x] I tested both happy and unhappy path of the functionality
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: keyless/pepper/service/src/main.rs [hunk 1]
```diff
@@ -85,6 +86,12 @@ async fn main() {
         Duration::from_secs(10),
     );
 
+    jwk::start_jwk_refresh_loop(
```
Comment (alinush) [inline keyless/pepper/service/src/main.rs:89]: Sweet!

**Review by alinush — APPROVED** [review]

**Review by zjma — COMMENTED** [review]
this allows a git commit to affect running-in-production service directly, very high risk :)

**Comment by zjma** [comment]
> this allows a git commit to affect running-in-production service directly, very high risk :)

Imagine you want to update the test jwk. With this setting you can't even test it locally...

**Comment by zjma** [comment]
I believe hardcoding it in the binary is still better. We can still have single source of truth by loading the json file on start.

I have an incoming refactoring that introduces an aptos-jwk-utils crate that should make it cleaner.

File: keyless/pepper/service/src/main.rs [hunk 2]
```diff
@@ -85,6 +86,12 @@ async fn main() {
...
+        &get_sample_iss(),
```
Comment (alinush) [inline keyless/pepper/service/src/main.rs:90]: @zjma, this isolates the impact of a malicious JSON file to only the test `iss`. Isn't this fine?

(Trying to address your concern.)

**Comment by alinush** [comment]
> > this allows a git commit to affect running-in-production service directly, very high risk :)
> 
> Imagine you want to update the test jwk. With this setting you can't even test it locally...

Mmmm... I guess you'd have to modify the code, yes. Previously, that was also the case.

Are you proposing another `include_str!` then? But a recompile will still be required in that case too when the JWK needs to change, no?

**Comment by zjma** [comment]
> > > this allows a git commit to affect running-in-production service directly, very high risk :)
> > 
> > 
> > Imagine you want to update the test jwk. With this setting you can't even test it locally...
> 
> Mmmm... I guess you'd have to modify the code, yes. Previously, that was also the case.
> 
> Are you proposing another `include_str!` then? But a recompile will still be required in that case too when the JWK needs to change, no?

yeah I’d go with an include_str

**Review by zjma — APPROVED** [review]

**Pull request merged**